exercises/series/src/test/java/SeriesTest.java

```diff
@@ -0,0 +1,144 @@
+
+import java.util.Arrays;
+import java.util.List;
+import junit.framework.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+
+public class SeriesTest {
+
+    @Test
+    public void hasDigitsShort() {
+        Series sut = new Series("01234");
+        List<Integer> expected = Arrays.asList(0, 1, 2, 3, 4);
+        List<Integer> actual = sut.getDigits();
+        assertNotNull(actual);
+        assertFalse(actual.isEmpty());
+        assertEquals(expected, actual);
+    }
+
+    @Test
```

Comment:
There's no instruction that would have prompted you to do so, but...

Please `@Ignore` all but the first test.  Doing so makes for a much more pleasant user experience when practitioners run the tests.  Without it, she would get a wall of text from all the failures.